src/contracts/Protocol/modifiers/NFTLocked.sol

```diff
@@ -5,6 +5,8 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/interfaces/IERC721.sol";
 
 abstract contract NFTLocked is Ownable {
+    bytes32 public constant PROGRAM_MANAGER_ROLE = keccak256("PROGRAM_MANAGER_ROLE");
```

Comment:
I wonder if there's a better place to stash these?  for now this will work - but as we add a ton of em, maybe in the file that uses the role?  or maybe in the nft that defines the role?